nemo/nemo/backends/pytorch/actions.py

```diff
@@ -97,7 +100,7 @@ def __get_top_sorted_modules_and_dataloader(self, hook):
           in DAG
 
         Returns:
-          list of modules with their call arguments and outputs, and dataset
```

Comment:
I disagree with the changed comments.
module should be pluralize into modules whenever necessary.

Reply:
sorry, this was done by wrong docstring re-factoring. will fix back